### PR TITLE
Creating retrofit

### PR DESCRIPTION
**Pull request by mikimik01**

### CODE REVIEW

**Comment by piotrgomola** [comment]
You forgot to add .gitignore to the repo - try to do it (there are some default ones for Android project). It should remove .gragle and .idea at least

File: app/src/main/java/com/example/footballmanager/network/FootballApiService.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+private val retrofit = Retrofit.Builder()
+    .baseUrl(BASE_URL)
+    .addConverterFactory(GsonConverterFactory.create(GsonBuilder().create()))
+    .build()
+
+
+interface FootballApiService {
+    @Headers(
+        "x-rapidapi-host: v3.football.api-sports.io",
+        "x-rapidapi-key: 182fc0d02721df1eca6a1ad83cf310ac")
```
Comment (piotrgomola) [inline app/src/main/java/com/example/footballmanager/network/FootballApiService.kt]: try to use local.properties for that:
https://medium.com/make-android/how-to-safeguard-your-api-keys-in-android-projects-with-github-secrets-5679e0e89a77

File: app/src/main/java/com/example/footballmanager/network/Fixture.kt
Comment (klaudiakow) [inline app/src/main/java/com/example/footballmanager/network/Fixture.kt:1]: keep each data class in separate file - it's a good practice which allows to keep codebase clean

File: app/src/main/java/com/example/footballmanager/network/Fixture.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package com.example.footballmanager.network
+
+import com.google.gson.annotations.SerializedName
+
+data class FixtureDataWrapper(
+    @SerializedName("response"   ) var responseBody   : ArrayList<ResponseBody> = arrayListOf()
+)
+
+data class ResponseBody(
+    @SerializedName("teams"   ) var teams   : Teams?   = Teams(),
```
Comment (klaudiakow) [inline app/src/main/java/com/example/footballmanager/network/Fixture.kt]: Is there something wrong with formatting?

File: app/src/main/java/com/example/footballmanager/network/FootballApiService.kt [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    @Headers(
+        "x-rapidapi-host: ${BuildConfig.RAPIDAPI_HOST}",
+        "x-rapidapi-key: ${BuildConfig.RAPIDAPI_KEY}")
+    @GET("fixtures")
+    fun getFixtures(
+        @Query("last") last: Int //2 characters
+    ):Call<FixtureDataWrapper>
+}
+
+object FootballApi{
```
Comment (klaudiakow) [inline app/src/main/java/com/example/footballmanager/network/FootballApiService.kt]: move it to separate file

File: app/src/main/java/com/example/footballmanager/network/FootballApiService.kt [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "https://v3.football.api-sports.io"
+
+private val retrofit = Retrofit.Builder()
+    .baseUrl(BASE_URL)
+    .addConverterFactory(GsonConverterFactory.create(GsonBuilder().create()))
+    .build()
+
+interface FootballApiService {
+    @Headers(
+        "x-rapidapi-host: ${BuildConfig.RAPIDAPI_HOST}",
```
Comment (klaudiakow) [inline app/src/main/java/com/example/footballmanager/network/FootballApiService.kt]: consider using interceptor for adding header

**Comment by klaudiakow** [comment]
In the future please remember about commit naming convension
Overall looks good! :)

**Review by klaudiakow — APPROVED** [review]

**Pull request merged**